packages/wp-now/src/wp-now.ts

```diff
@@ -215,13 +215,15 @@ async function runWordPressMode(
 	php: NodePHP,
 	{ documentRoot, projectPath, absoluteUrl }: WPNowOptions
 ) {
-	await initWordPress(
-		php,
-		'user-provided',
-		projectPath,
-		documentRoot,
-		absoluteUrl
-	);
+	php.mount(projectPath, documentRoot);
+	if (!php.fileExists(`${documentRoot}/wp-config.php`)) {
+		await initWordPress(
+			php,
+			'user-provided',
+			documentRoot,
+			absoluteUrl
+		);
+	}
```

Comment:
In the future, here we could handle another case too e.g. validate if `wp-config.php` is valid for the `wp-env` stack, propose to recreate it, etc.